src/Symfony/Component/Form/Button.php

```diff
@@ -375,14 +355,7 @@ public function handleRequest($request = null)
     }
 
     /**
-     * Submits data to the button.
-     *
-     * @param string|null $submittedData Not used
-     * @param bool        $clearMissing  Not used
-     *
-     * @return $this
-     *
-     * @throws Exception\AlreadySubmittedException if the button has already been submitted
+     * {@inheritdoc}
```

Comment:
@Tobion , can I leave this change? I just removed the almost duplicating phpdoc.